tests/events/programs/events/src/lib.rs

```diff
@@ -31,6 +44,14 @@ pub struct Initialize {}
 #[derive(Accounts)]
 pub struct TestEvent {}
 
+#[derive(Accounts)]
+pub struct TestEventCpi<'info> {
+    program: Program<'info, crate::program::Events>,
+    /// CHECK: this is the global event authority
+    #[account(seeds=[b"__event_authority"], bump)]
```

Comment:
Specifying these accounts for all instructions in a program is going to be really cumbersome especially with `__event_authority` as seed. `__` is used for internal usage but we are exposing it to outside world here. And also the seed cannot be changed because that's being used inside the macro to sign the transaction anyway.

Reply:
Yeah I agree this is a huge PITA. I'm looking forward to seeing some new ideas on this :)